### ISSUE TITLE
Is this Role Abandoned?

### ISSUE THREAD

**Issue by jamiejackson** [gsoft-inc/ansible-role-azure-devops-agent#82]
I started making updates to get it to work but I just realized that there are a bunch of open PRs already. Is this role still supported? If not, what are people using instead?

Attn: @yohanb , @huypn12, @juju4, @Poil, @DanSibbernsen, et al.

**Comment by Poil** [comment]
Hi,

I think we should create a new role to install agent v3 that is cleaner and need less hack on recent system.

**Comment by mlgitdev** [comment]
yeah, bunch of PRs, some are useful, the code should be updated to handle the syntax, other stuff are ok and vars can be used to define the correct download package url, I am ok wit h most of the hacks.
